.circleci/continue_config.yml

```diff
@@ -15,6 +15,10 @@ parameters:
   nightly-run:
     type: string
     default: "inactive"
+  integration-type:
+    type: enum
+    default: "*"
+    enum: [ "*", "airflow", "dagster", "dbt", "publish", "python" ]
```

Comment:
Why is "publish" an integration type?

Reply:
enum content is just copied from another config.